hazelcast/src/main/java/com/hazelcast/internal/partition/ChunkSerDeHelper.java

```diff
@@ -46,10 +46,14 @@
 
     public ChunkSerDeHelper(ILogger logger, int partitionId,
                             Collection<ChunkSupplier> chunkSuppliers,
+                            boolean chunkedMigrationEnabled,
                             int maxTotalChunkedDataInBytes) {
         assert chunkSuppliers != null;
         assert logger != null;
-        assert maxTotalChunkedDataInBytes > 0 : maxTotalChunkedDataInBytes;
+
+        if (chunkedMigrationEnabled) {
+            assert maxTotalChunkedDataInBytes > 0 : maxTotalChunkedDataInBytes;
```

Comment:
maybe `assert !chunkedMigrationEnabled || (maxTotalChunkedDataInBytes > 0) : maxTotalChunkedDataInBytes;` to eliminate the `if` statement?

Reply:
done